Disable editing on closed accounts

dospuntos opened the pull request:
This fixes #51 , by disabling the fields for entering transactions if the account is closed.
Double-click on a transaction for editing will show a warning message. 

![image](https://github.com/HaikuArchives/CapitalBe/assets/12958546/8d888837-369d-46bf-840e-5a5fb801e5e3)

humdingerb commented on src/CheckView.cpp:
```diff
@@ -294,6 +294,8 @@ CheckView::HandleNotify(const uint64& value, const BMessage* msg)
 				}
 			}
 		}
+	} else if (value & WATCH_CHANGE) {
```
Not sure how strict or if you apply the Haiku coding style in this project.
The curly brackets aren't needed for the single-line statement.

Otherwise the change looks good!

dospuntos replied:
I'll remove them. I also forgot to generate the catkeys file... Will update the PR in a minute

dospuntos replied:
@humdingerb `git-haiku-format` added the curly brackets back, so I think I'll just leave them there.
I generated the catkeys, and also found a few other cases where closed accounts should be taken into account (editing account settings and transfer between accounts).

humdingerb replied:
Weird. I thought haiku-format did respect hat part of the coding style... No matter.

dospuntos commented:
@humdingerb Unless you have any suggestions for changes to the wording of the messages, I think this project can be added to Polyglot now.

humdingerb commented:
> @humdingerb Unless you have any suggestions for changes to the wording of the messages, I think this project can be added to Polyglot now.

The only thing I caught was that the ellipsis of the menu item "Transaction|Delete..." should be removed, because the command does instantly remove the currently selected transaction.

After that I can upload the latest en.catkeys to Polyglot.

dospuntos commented:
> The only thing I caught was that the ellipsis of the menu item "Transaction|Delete..." should be removed, because the command does instantly remove the currently selected transaction.

Updated. I also removed the ellipsis from "File | Scheduled transactions | Remove".

Maybe there should be a confirmation dialog before deleting transactions and scheduled transactions? It would be safer, but it can also be annoying with too many dialogs. Currently, only deleting accounts require confirmation.

humdingerb commented:
> Maybe there should be a confirmation dialog before deleting transactions and scheduled transactions? It would be safer, but it can also be annoying with too many dialogs. Currently, only deleting accounts require confirmation.

Best would be an (unlimited) undo/redo feature... :)

Let's merge this then and I'll upload the catkeys and helpfiles to Polyglot.
